src/app/game/game-multiplayer/services/web-socket.service.ts

```diff
@@ -59,6 +59,8 @@ export class WebSocketService {
     });
   }
 
+  // Disable any type check for only this one because the spread parameter will take parameters of different types
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   emit(eventName: string, ...rest: any) {
```

Comment:
Still want to explore if its possible to make this one work without ignoring it, but maybe thats because i havent gone deep enough into this yet.